deploy/templates/FhirSynapsePipelineTemplate.json

```diff
@@ -55,7 +55,7 @@
         },
         "packageUrl": {
             "type": "string",
-            "defaultValue": "https://yufeiadlsv2.blob.core.windows.net/fhir/publish_1124.zip",
+            "defaultValue": "https://fhirdeploy.blob.core.windows.net/fhir/Microsoft.Health.Fhir.Synapse.FunctionApp.zip",
```

Comment:
This is a public accessible private build. We still need to update it to the formal one when release.